### PR TITLE
Plot, save files and adapt examples to constraints and objectives.

### PR DESCRIPTION
**Pull request by tgouss**

### CODE REVIEW

File: biorbd_optim/optimal_control_program.py [hunk 1]
```diff
@@ -290,5 +296,40 @@ def solve(self):
...
+
+    @staticmethod
+    def load(biorbd_model_path, name):
+        with open(name, "rb") as file:
+            data = pickle.load(file)
+            ocp = data["ocp"]
+            sol = data["sol"]
+            for nlp in ocp.nlp:
+                nlp["model"] = biorbd.Model(biorbd_model_path)
+        return (ocp, sol)
```
Comment (pariterre) [inline biorbd_optim/optimal_control_program.py:335]: Can you add a test that compares the results before and after loading?

File: biorbd_optim/plot.py [hunk 2]
```diff
@@ -204,52 +204,89 @@ def graphs(self):
...
+    #             rtol=1e-6,
+    #         )
+    #
+    #         if i < len(t) - 2:
+    #             integrated_t = np.concatenate(integrated_t, integration.t[:-1])
+    #             integrated_state = np.concatenate((integrated_state, integration.y[:, :-1]), axis=1)
+    #         else:
+    #             integrated_t = np.concatenate(integrated_t, integration.t)
+    #             integrated_state = np.concatenate((integrated_state, integration.y), axis=1)
+    #     return integrated_t, integrated_state
```
Comment (pariterre) [inline biorbd_optim/plot.py:281]: Is that a [WIP] PR?

File: biorbd_optim/optimal_control_program.py (hunk 1 above)
Comment (tgouss) [reply biorbd_optim/optimal_control_program.py:335]: test ? in "load_eocar.py" or elsewhere ?

Comment (pariterre) [reply biorbd_optim/optimal_control_program.py:335]: in the "tests" folder

Comment (pariterre) [reply biorbd_optim/optimal_control_program.py:335]: Is this done? I would close this PR if it is

**Pull request merged**